src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ContainerDebugCommandTest.php

```diff
@@ -136,6 +136,88 @@ public function testDescribeEnvVar()
         $this->assertStringContainsString(file_get_contents(__DIR__.'/Fixtures/describe_env_vars.txt'), $tester->getDisplay(true));
     }
 
+    public function testGetDeprecation()
+    {
+        static::bootKernel(['test_case' => 'ContainerDebug', 'root_config' => 'config.yml', 'debug' => true]);
+        $path = sprintf('%s/%sDeprecations.log', static::$kernel->getContainer()->getParameter('kernel.cache_dir'), static::$kernel->getContainer()->getParameter('kernel.container_class'));
+        touch($path);
+        file_put_contents($path, serialize([
+            ['type' => 16384,
+             'message' => 'The "Symfony\Bundle\FrameworkBundle\Controller\Controller" class is deprecated since Symfony 4.2, use Symfony\Bundle\FrameworkBundle\Controller\AbstractController instead.',
+             'file' => '/home/hamza/projet/contrib/sf/vendor/symfony/framework-bundle/Controller/Controller.php',
+             'line' => 17,
+             'trace' => [
+                 [
+                         'file' => '/home/hamza/projet/contrib/sf/src/Controller/DefaultController.php',
+                         'line' => 9,
+                         'function' => 'spl_autoload_call',
+                 ],
+             ], 'count' => 1, ], ]));
```

Comment:
the formatting/indenting looks a bit odd here :/

Reply:
I didn't really know how to indent it :p, should I inline everything ? or just let the count down ?

Reply:
i'd say

```php
file_put_contents($path, serialize([[
    'type' => 16384,
    'message' => 'The "Symfony\Bundle\FrameworkBundle\Controller\Controller" class is deprecated since Symfony 4.2, use Symfony\Bundle\FrameworkBundle\Controller\AbstractController instead.',
    'file' => '/home/hamza/projet/contrib/sf/vendor/symfony/framework-bundle/Controller/Controller.php',
    'line' => 17,
    'trace' => [[
        'file' => '/home/hamza/projet/contrib/sf/src/Controller/DefaultController.php',
        'line' => 9,
        'function' => 'spl_autoload_call',
    ]],
    'count' => 1
]]));
```

Reply:
done